crates/blockifier/src/transaction/account_transaction.rs

```diff
@@ -191,6 +197,14 @@ impl AccountTransaction {
         Ok(())
     }
 
+    pub fn get_transaction_hash(&self) -> &TransactionHash {
```

Comment:
Don't name getters methods using the `get_` prefix

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter